astropy/io/misc/asdf/tags/transform/tests/test_transform.py

```diff
@@ -52,8 +52,41 @@ def custom_and_analytical_inverse():
     astmodels.RotateCelestial2Native(5.63*u.deg, -72.5*u.deg, 180*u.deg),
     astmodels.RotationSequence3D([1.2, 2.3, 3.4, .3], 'xyzx'),
     astmodels.SphericalRotationSequence([1.2, 2.3, 3.4, .3], 'xyzy'),
-    custom_and_analytical_inverse(),
-]
+    astmodels.AiryDisk2D(10., 0.5, 1.5),
```

Comment:
People can specify parameter values as keyword arguments, right?  It might be a good idea to also construct them that way, so we can protect that aspect of the public interface.

Reply:
Yeah - the existing tests didn't do it this way so I just stuck with supplying the arguments like that, but I can change them all to use keyword arguments.